requirements/requirements.in

```diff
@@ -31,7 +31,7 @@ billiard!=3.5.0.5  # Should be resolved in celery 4.3: https://github.com/celery
 mock==2.0.0
 Pillow>=6.2.2,<7.0.0
 phonenumberslite~=8.8
-psycopg2~=2.7.7
+psycopg2>=2.8.4  # Python 3.8 support
```

Comment:
Actually, this does install 2.8.5, because it is the "best" version that satisfies ">=2.8.4".

I thought it was safe not to specify a max version (as we do for Pillow on line 32) because the name "psycopg**2**" makes me think that it's unlikely to suddenly drop a "3.0.0" that breaks stuff. But we could change this to
```suggestion
psycopg2>=2.8.4,<3.0.0  # Python 3.8 support
```

Reply:
> Actually, this does install 2.8.5, because it is the "best" version that satisfies ">=2.8.4".

Of course. Ignore my comment.

> I thought it was safe not to specify a max version...

Yeah, don't bother with a max version unless/until we know there is a problem with that. Keep it simple.